Utils/azure-explorer-common/test/java/com/microsoft/tooling/IntegrationTestBase.java

```diff
@@ -284,28 +305,52 @@ private String applyRegex(String text) {
         }
         return text;
     }
-    
+
     private static Boolean IsMocked() {
-        // TODO: support un mocked mode
+        String keyValue=System.getProperty("isMockedCase");
+        if (keyValue!=null&&keyValue.equalsIgnoreCase("false"))
+            return false;
         return true;
     }
     
+    private synchronized Response redirectErrorHost(Interceptor.Chain chain) throws IOException {
+        Request request=chain.request();
+        String url=chain.request().url().toString();
+        if(IS_MOCKED) {
+            url=url.replace(MOCK_URI, "https://management.notexistazure.com");
+        }
+        else {
+        url=url.replace(GLOBAL_ENDPOINT, "https://management.notexistazure.com");
+        }
+        Request newRequest=request.newBuilder().url(url).build();
+        return chain.proceed(newRequest);
+    
+    }
+    //get auth file for nonmock case
+    //-DisMockedCase=false -DauthFilePath="c:\config.azureauth"
+    private static String getAuthFile() {
+        String authFilePath=System.getProperty("authFilePath");
```

Comment:
Yes, it will return empty string. It will be caught in below code line 124.
            try {
                credentials = ApplicationTokenCredentials.fromFile(new File(azureAuthFile));
            } catch (Exception e) {
                throw new Exception("Fail to open auth file:" + azureAuthFile);
            }